defaults/main.yml

```diff
@@ -38,6 +38,7 @@ openwisp2_time_zone: "UTC"
 openwisp2_context: {}
 openwisp2_allowed_hosts: []
 openwisp2_leaflet_config: {}
+openwisp2_geocoding_check: true
```

Comment:
Please update this variable in [README](https://github.com/openwisp/ansible-openwisp2#role-variables)

Reply:
Sure